modules/installation-aws-tested-machine-types.adoc

```diff
@@ -41,7 +41,9 @@ ifdef::localzone[]
 .Machine types based on x86_64 architecture for AWS Local Zones
 [%collapsible]
 ====
+* `c5d.2xlarge`
 * `m5.xlarge`
+* `t3.xlarge`
```

Comment:
@yunjiang29 Please take a look into these items, considering my last comment here[1]

[1] https://github.com/openshift/openshift-docs/pull/54535#discussion_r1071397179

Reply:
@mjpytlak Agree with @mtulio [1], please add `m5.2xlarge` and `c5.2xlarge`, thanks. 

[1] https://github.com/openshift/openshift-docs/pull/54535#discussion_r1071709418